index.js

```diff
@@ -58,7 +58,15 @@ class MobileFriendly {
       console.log(`Checking site ${this.url} ...`);
 
       request.post(options, (err, data) => {
-        const body = JSON.parse(data.body);
+        let body;
```

Comment:
The err argument of the callback is unfortunately null, and data contains the html code of the 502 error page. That's why i check also the err object before i overwrite it to return the err from the API if it will be available.